source/mainnet/docs/smart-contracts/guides/invoke-instance.rst

```diff
@@ -1,8 +1,8 @@
 .. _invoke-instance:
 
-================================
-Invoke a smart contract instance
-================================
+======
+Invoke
```

Comment:
invoke instance maybe?

Reply:
It's more consistent if all titles in "Deploy" have 1 word per name